ProcessMaker/processmaker
# Upgrade to Laravel 7.x

Issues
https://processmaker.atlassian.net/browse/FOUR-6564
https://processmaker.atlassian.net/browse/FOUR-6565

Upgrade to Laravel 7

Package PRs
- https://github.com/ProcessMaker/package-files/pull/86
- https://github.com/ProcessMaker/package-collections/pull/326
- https://github.com/ProcessMaker/package-data-sources/pull/241
- https://github.com/ProcessMaker/package-comments/pull/82
- https://github.com/ProcessMaker/package-versions/pull/54
- https://github.com/ProcessMaker/package-savedsearch/pull/327

Note: PMQL was locked to version <=6 so I temporarily specified `dev-develop` until there's a new release. I would recommend doing a release then updating this composer.json, then running composer update and committing composer.lock, before merging this PR.

Testing:
- Install all packages and use the `feature/FOUR-6565` branch for the above packages.
- Skim the code changes and identify areas that may be affected
- Try out a bunch of different features of the app

_Original Laravel Shift PR text below_

This pull request includes the changes for upgrading to Laravel 7.x. Feel free to commit any additional changes to the `shift-69390` branch.

**Before merging**, you need to:

- Checkout the `shift-69390` branch
- Review **all** pull request comments for additional changes
- Run `composer update` (if the scripts fail, add `--no-scripts`)
- Clear any config, route, or view cache
- Thoroughly test your application ([no tests?](https://laravelshift.com/laravel-test-generator), [no CI?](https://laravelshift.com/ci-generator))

If you need help with your upgrade, check out the [Human Shifts](https://laravelshift.com/human-shifts). You may also join the [Shifty Coders](https://laravelshift.com/shifty-coders) Slack workspace to level-up your Laravel skills.
